projects.md

```diff
@@ -8,6 +8,49 @@ by other users within the community. As such, we're happy to curate a list of pr
 them visibility to a larger audience. To have you project listed here, send a request to the
 [developer's mailing list](mailto:dev@accumulo.apache.org)
 
+## Open source projects using Accumulo
+
+#### Fluo
+
+[Fluo](https://fluo.apache.org) builds on Accumulo and enables low latency, continuous incremental processing of big data.
+
+#### Geomesa
+
+[Geomesa](http://www.geomesa.org/) is an open-source, distributed, spatio-temporal database built on a number of distributed cloud data storage systems, including Accumulo, HBase, Cassandra, and Kafka.
+
+#### Geowave
+
+[Geowave](https://ngageoint.github.io/geowave/) is a library for storage, index, and search of multi-dimensional data on top of a sorted key-value datastore. 
+
+#### Gora
+
+[Gora](https://gora.apache.org/) open source framework provides an in-memory data model and persistence for big data.  Accumulo's continuous ingest test suite was adapted to Gora and called [Goraci](http://gora.apache.org/current/index.html#goraci-integration-testsing-suite).
+
+#### Graphulo
+
+Graphulo is a Java library for Apache Accumulo which delivers server-side sparse matrix math primitives that
+enable higher-level graph algorithms and analytics. [Code](https://github.com/Accla/graphulo).
+
+#### Hive
+
+[Hive](https://hive.apache.org/) data warehouse software facilitates reading, writing, and managing large datasets residing in distributed storage using SQL.  Hive has [handlers for Accumulo](https://cwiki.apache.org/confluence/display/Hive/AccumuloIntegration).
```

Comment:
Suggest: "Hive has the ability to read and write data in Accumulo using the [AccumuloStorageHandler](https://cwiki.apache.org/confluence/display/Hive/AccumuloIntegration)"